### ISSUE TITLE
Autocomplete Does Not Work With community.general >= 5.0.0

### ISSUE THREAD

**Issue by kjoyce77** [ansible/ansible-language-server#434]
### Summary

TLDR: When I use community.general 4.8.6 syntax autocomplete works, when I used 5.5.0 it does not.

To elaborate...

With 5.5.0 I do get a prompt that says to use 'community.general.system.sudoers' for example.  But when I do that I still don't get autocomplete and the help popups.

I looked at the repos and it appears 5.0.0 removed all the symbolic links in the plugins/modules directory.  Since in 4.8.6 I used 'community.general.sudoers' and all works well I presume something is going wrong with using FQCNs with more than 3 path elements.

### Extension version

v0.13.59

### VS Code version

1.71.0

### Ansible Version

```shell
ansible [core 2.12.1]
  config file = /home/kjoyce/projects/cloud_server/cloud-server/ansible.cfg
  configured module search path = ['/home/kjoyce/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/kjoyce/projects/cloud_server/.venv/lib/python3.10/site-packages/ansible
  ansible collection location = /home/kjoyce/projects/cloud_server/.venv/usr/share/ansible/collections/ansible_collections
  executable location = /home/kjoyce/projects/cloud_server/.venv/bin/ansible
  python version = 3.10.0 (default, Oct 24 2021, 15:06:41) [GCC 7.5.0]
  jinja version = 3.0.3
  libyaml = True
```

### OS / Environment

Windows 10, remote SSH Ubuntu 18.04, ansible-lint 5.3.1

**Comment by ssbarnea** [comment]
@priyamsahoo I got some feedback from @bcoca and basically that would be logic:

Basically symlinks and aliases from `meta/runtime.yml` can be considered as compatibility aliases. The recommended / normalized values should be the target/resolved/longer longer names, as these would also be faster. When migrations happen and code will be moved, but not without using either symlinks or aliases, making the "resolved" version as always better to use.

I already started working on fixing the linter to accept these.

**Comment by felixfontein** [comment]
@kjoyce77 please note that you should **NOT** use `community.general.system.sudoers`. Use `community.general.sudoers`. The long name is wrong and can likely stop working eventually without any warning.

**Comment by kjoyce77** [comment]
> @kjoyce77 please note that you should **NOT** use `community.general.system.sudoers`. Use `community.general.sudoers`. The long name is wrong and can likely stop working eventually without any warning.

OK, thank you, but that does not address the issue.  To make things clearer here a screenshots of what community.general 4.8.7 autocompletes look like vs. 5.7.0.  I only mentioned the longer name because the mouseover told me to use the longer name when I use 5.7.0.
![community general 4 8 7](https://user-images.githubusercontent.com/22777125/195222777-08049b7c-1a96-4708-b305-5adf51c6eaf0.png)
![community general 5 7 0](https://user-images.githubusercontent.com/22777125/195222782-1ff384be-199c-46ce-8764-81d9a5099504.png)

**Comment by felixfontein** [comment]
@kjoyce77 that's a bug in the VScode extension / language server which hopefully will get eventually fixed. See https://github.com/ansible/vscode-ansible/issues/573.
